chapter_01/hinglish/introduction.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<p>not clear.. (LED light aapki clipboard par flicker??)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/abhiabhi94/learn-python/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>yes, it should have been <code>agar aap chahte hai LED light aapki keyboard par flicker...</code>	. Does this seem reasonable?</p>

Reply:
<p>maybe change this to something like <code>produce a sound</code>  which could possibly me more relatable.</p>

Reply:
<p>yeah</p>